go/parquet/compress/compress.go

```diff
@@ -92,6 +92,10 @@ type Codec interface {
 
 var codecs = map[Compression]Codec{}
 
+func RegisterCodec(compression Compression, codec Codec) {
+	codecs[compression] = codec
+}
```

Comment:
Can you add godoc comments to this?

Reply:
Thanks! Added some basic comment and simple usage example